Automattic/wp-calypso
# Remove unused e2e test steps for domain mapping

#### Changes proposed in this Pull Request

* Clean up unused test steps as we don't go through checkout when mapping a domain for users with plan - see #52297 .

#### Testing instructions

* Technically nothin should change as the test steps were already skipped. Anyway - verify that e2e tests completed successfully
